django/contrib/admin/templates/admin/change_form.html

```diff
@@ -79,9 +79,19 @@
                     e.preventDefault();
                     showRelatedObjectLookupPopup(this);
                 });
+                $('body').on('click', '.related-widget-wrapper-link', function(e) {
+                    e.preventDefault();
+                    if (this.href) {
+                        showRelatedObjectPopup(this);
+                    }
+                });
+                $('body').on('change', '.related-widget-wrapper select', updateRelatedObjectLinks);
+                $('.related-widget-wrapper select').trigger('change');
```

Comment:
to replace this line in the old code:
```
container.find('.related-widget-wrapper select').each(updateLinks);
```
Haven't double checked it is required / makes sense.